mathics/builtin/files_io/files.py

```diff
@@ -528,29 +502,9 @@ class OpenRead(_OpenAction):
 
     >> OpenRead["ExampleData/EinsteinSzilLetter.txt", CharacterEncoding->"UTF8"]
      = InputStream[...]
-    #> Close[%];
+    >> Close[%];
 
     S> Close[OpenRead["https://raw.githubusercontent.com/Mathics3/mathics-core/master/README.rst"]];
```

Comment:
I keep these sandbox tests for another round, because I need to figure out how to mark them in the pytests. Any idea?

Reply:
I don't have a thought right now about sandboxed tests. 

As for `#> Close[%];` vs. `>> Close[%];`,  if it is instructive that is okay. For example: 

```mathematica
>> OpenRead["ExampleData/EinsteinSzilLetter.txt", CharacterEncoding->"UTF8"]
     = InputStream[...]

Streams should be closed when done with them: 
 >> Close[%];
```

But if it is not, but just done to make testing more robust and for bookkeeping keep the `#>` so we don't pollute the docs.

Reply:
Good: I leave then `S>`for another round. Also, I have reverted  `#> Close` ->`>> Close` when it is not instructive.

Reply:
In Sphinx, the cleanup is implemented in a special code block directive `testcleanup`:
https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html